docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
+* When I audit our systems, I want an easy way to verify the New Relic configuration is defined as expected
+
+**New Relic CLI**
+
+* When I deploy an applications, I want to record the deployment in New Relic without having to learn how their API works
+* When service ownership changes, I want an easily, automated way to change tags within New Relic
+
+**New Relic Client**
+
+* When I have custom tooling in place that I want to leverage New Relic, I want to be able to use a supported library.
+* When I build a deployment pipeline using serverless functions, I want to include New Relic.
+
+
+
+## Developer Toolkit v1
+
+To solve for the above problems, the initial version of the Developer Toolkit is defined to contain the following resources.  All resources shall be built as Open Source projects, and actively engage with the broader community.
+
+* **[New Relic Terraform Provider](https://github.com/terraform-providers/terraform-provider-newrelic/)** - The New Relic Terraform Provider enables Observability as Code, reducing developer toil, and allows users to manage their entire ecosystem in a single place.
+
+   * Built on the New Relic Client
+   * Fully implementations APIs available through the New Relic Client
+   <br/>
+
+* **New Relic Client** - The New Relic Client provides the building blocks for many tools in the toolkit, enabling quick access to the suite of New Relic APIs.  As a library, it can also be leveraged within your own custom applications.
+
+   * Full support for the followin New Relic API endpoints
+     * APM REST v2 API
+     * Infrastructure API
+     * Synthetics API
+     * GraphQL
+   <br/>
+
+* **New Relic CLI** - The New Relic CLI allows integrating New Relic into your existing workflows. Be it fetching data from your laptop while troubleshooting an issue, or adding New Relic into your CI/CD pipeline.
+
+   * Build on the New Relic Client
+   * Initial implementation of
+     * Deployment Markers
+     * Tagging
+   <br/>
+
+* **[New Relic AWS CloudFormation Integration](https://github.com/newrelic/cloudformation-partner-integration)** - The AWS CloudFormation Integration enables developers using CloudFormation to easily provision a subset of New Relic resources.
+
+   * Build on the NR Client
+   * Initial implementation of
+     * Basic NRQL Alert configuration
+   <br/>
+
+
+## Milestones
+
+Ongoing work will be tracked in the [Developer Toolkit](https://github.com/orgs/newrelic/projects/6) Project, which will link to each individual tool that is in flight.  Here the latest summary for planned work by the New Relic team:
+
+### 2019 Winter
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Completed | New Relic Terraform Provider | (Tracked Internally)| The current Terraform Provider for New Relic was created by the community, and suffering from lack of attention.  DTK will integrate into the ecosystem, and create a clean base to work from. |
```

Comment:
change to "suffered from lack of attention" or "was suffering from lack of attention".  Also - do you want to use DTK when referring to the team or the suite of tools?  I think we shouldn't use this acronym in the open when referring to our internal team team.  So suggest you change to "We will integrate"..or "New Relic will integrate"